tests/generation/test_framework_agnostic.py

```diff
@@ -633,7 +633,7 @@ def test_eos_token_id_int_and_list_beam_search(self):
             "do_sample": False,
             "num_beams": 3,
         }
-        expectation = 13
+        expectation = 20
```

Comment:
```suggestion
        if is_pt:
            expectation = 20
        else:
            # TODO (joao): fix me
            expectation = 13
```

This test will likely fail on TF, since we haven't applied this upgrade there. Let's add a TODO for now